hw-model/c-binding/examples/Makefile

```diff
@@ -16,7 +16,7 @@
 EXTRA_CARGO_CONFIG = target.'cfg(all())'.rustflags = [\"-Dwarnings\"]
 OUT = out
 TARGET = $(OUT)/smoke_test
-RTL_SOC_IFC_INCLUDE_PATH = ../../../hw/latest/rtl/src/soc_ifc/rtl
+RTL_SOC_IFC_INCLUDE_PATH = ../../../hw/1.0/rtl/src/soc_ifc/rtl
```

Comment:
Should we leave this as "latest"? This c-binding is just for testing libcaliptra against the HW model. I think we'd want to continue to test it against the latest HW for future development?

Reply:
I've been consistent in this PR pointing everything at 1.0 by default, as latest isn't working yet (verilator, fpga, and sw-emulator hw-models don't support latest).

The expectation is to change these once they work :)

Reply:
Ah understood. This makes sense then. Thanks!